.github/workflows/ci.yml

```diff
@@ -62,6 +62,11 @@ jobs:
         env:
           CI: true
 
+      - name: Check api types
+        run: |
+          yarn generate-api-types
+          [[ -n $(git status --porcelain) ]] && echo "You need to run yarn generate-api-types" && exit 1 || true
```

Comment:
Assuming it'll catch lint tasks above, maybe let's change the message to include `or yarn lint`